cli/src/main/java/com/devonfw/tools/ide/url/model/file/UrlStatusFile.java

```diff
@@ -70,11 +70,23 @@ protected void doLoad() {
   @Override
   protected void doSave() {
 
-    try (BufferedWriter writer = Files.newBufferedWriter(getPath(), StandardOpenOption.CREATE)) {
+    try (BufferedWriter writer = Files.newBufferedWriter(getPath())) {
```

Comment:
This was an evil bug. If explicit `StandardOpenOperation`s are specified as argument, I usually think that the developer made some smart choice here. However, this was pure non-sense:
If the JSON file already existed, the new JSON was appended causing invalid JSON and reading the JSON diff in git made it very tricky to figure out what is actually going wrong here.